src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -1586,10 +1588,25 @@ def new_nwm_q0(run_results):
                 debuglevel,
             )
 
-            #q0 = run_results
-            q0 = new_nwm_q0(run_results)
+        loop.run_until_complete(asyncio.gather(execute_model, build_next_forcing))
+        run_results = execute_model.result()  # gives us whatever is returned, once it is done.
+        qlats, usgs_df = build_next_forcing.result()  # gives us whatever is returned, once it is done.
+
```

Comment:
Lines 1591-1594 appear to be a merge error; they should be superseded by the similar lines under the if statement starting on line 1598.